src/index.js

```diff
@@ -62,10 +75,12 @@ function transformToAddEventsMessage(cloudWatchMessage) {
  * @returns {Object}          Outgoing Scalyr message.
  */
 function transformToUploadLogsMessage(cloudWatchMessage) {
+  let serverHost = (defaultServerHost || `cloudwatch-${cloudWatchMessage.owner}`);
```

Comment:
'let' on these two lines should be 'const' (same reason as above, value is never changed)